src/filter/GaussianFilter.js

```diff
@@ -0,0 +1,58 @@
+import IJ from '../ij';
```

Comment:
We will have to consider the fact that an image may have an alpha channel for which the kernel should not be applied. Using the getDataXY and setDataXY and a loop `for (let i=0; i<this.components; i++)`
The difference between this.components and this .channels is that this.channels is the pixel array size (so 4 for RGBA) while components is related to the color model (so 3 for RGBA, 1 for grey or 4 for CMYK).